apps/ledger-live-mobile/src/families/celo/ActivateFlow/02-Summary.tsx

```diff
@@ -92,23 +89,26 @@ export default function ActivateSummary({ navigation, route }: Props) {
         recipient: chosenValidator?.address ?? "",
       }),
     );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [updateTransaction, bridge, setTransaction, chosenValidator]);
```

Comment:
Don't think we shouldn't have `transaction` as a dep here, so I guess it was a mistake :thinking_face: 
```suggestion
  }, [updateTransaction, bridge, setTransaction, chosenValidator, transaction]);
```